management/pom.xml

```diff
@@ -88,6 +120,19 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.1</version>
+                <configuration>
+                    <source>1.7</source>
```

Comment:
Actually, I think I'm going to keep it at 7.  Only issue is having my mac switch between 7 and 8 for console work and this work here.